src/utils/LowMemPeakPickerHiRes_RandomAccess.cpp

```diff
@@ -1,4 +1,4 @@
-// --------------------------------------------------------------------------
+LowMemPeakPickerHiRes_RandomAccess// --------------------------------------------------------------------------
```

Comment:
So there's two solutions:

1) Change the name of the tool
2) Change the schema (to be honest, I am not sure why the name regex is so
restrictive)

On Thu, Jul 14, 2016 at 8:57 PM Hannes Roest <notifications@github.com>
wrote:

> In src/utils/LowMemPeakPickerHiRes_RandomAccess.cpp
> <https://github.com/OpenMS/OpenMS/pull/2052#discussion_r70863404>:
>
> > @@ -1,4 +1,4 @@
> > -// --------------------------------------------------------------------------
> > +LowMemPeakPickerHiRes_RandomAccess// --------------------------------------------------------------------------
>
> thats probably not what we want...
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenMS/OpenMS/pull/2052/files/327b79e08ac957e43dd430d3627c17c6a7930e1c#r70863404>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABd0pQAvpcLFRlD6t0vJj_bwpV0CAP3Lks5qVoatgaJpZM4JMglY>
> .
>